tests/test_yaml_search_space/correct_config.yaml

```diff
@@ -0,0 +1,31 @@
+search_space:
+  learning_rate:
+    lower: 0.00001
+    upper: 0.1
+    log: TRUE
+    is_fidelity: off
+
+  num_epochs:
+    lower: -3
+    upper: 30
+    log: false
+    is_fidelity: on
```

Comment:
This looks a bit weird, could we have another parameter name for this ?

Reply:
parameter naming is now fixed [here](https://github.com/automl/neps/pull/46/commits/15d4cc885e44f673eaaa6fd2f1b1e16ec6ffaae2)